lxd/main.go

```diff
@@ -721,7 +721,7 @@ func setupLXD() error {
 		}
 
 		if askBool("Would you like LXD to be available over the network (yes/no)? ") {
-			networkAddress = askString("Address to bind LXD to: ")
+			networkAddress = askString("Address to bind LXD to (not including port): ")
 			networkPort = askInt("Port to bind LXD to: ", 1, 65535)
```

Comment:
On Tue, Mar 22, 2016 at 12:55:29AM -0700, Stéphane Graber wrote:
> > @@ -721,7 +721,7 @@ func setupLXD() error {
> >  		}
> >  
> >  		if askBool("Would you like LXD to be available over the network (yes/no)? ") {
> > -			networkAddress = askString("Address to bind LXD to: ")
> > +			networkAddress = askString("Address to bind LXD to (not including port): ")
> >  			networkPort = askInt("Port to bind LXD to: ", 1, 65535)
> 
> We should change string to say that 8443 is recommended.

Done.  I'd also like to say for the trust password "enter means no password",
but then that begs for a longer description of what that means...